sdk/storage/storage-blob/samples/javascript/azureAdAuth.js

```diff
@@ -3,6 +3,7 @@
 
 /*
   ONLY AVAILABLE IN NODE.JS RUNTIME
+  If you are using the browser, you can use the InteractiveBrowserCredential provided via @azure/identity or any other feasible implementation of TokenCredential.
```

Comment:
```suggestion
  If you are using the browser, you can use the InteractiveBrowserCredential provided via @azure/identity or any other implementation of TokenCredential.
```

Reply:
The other kinds of TokenCredential implementation need to be available for browser. That's what "feasible" is for.